pycroft/lib/membership.py

```diff
@@ -200,3 +204,25 @@ def user_memberships_query(user_id: int, active_groups_only: bool = False) -> Qu
             .group_by(Membership.id)
     )
     return memberships
+
+
+def change_membership_active_during(
+        membership_id: int,
+        begins_at: datetime,
+        ends_at: Optional[datetime],
+        processor: User
+    ):
```

Comment:
The „missing return type annotation“ mypy complains about means adding `-> None` to signify that nothing is returned.
Together with more [`black`-friendly styling](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#how-black-wraps-lines) this would amount to
```suggestion
def change_membership_active_during(
    membership_id: int,
    begins_at: datetime,
    ends_at: Optional[datetime],
    processor: User,
) -> None:
```